cpp/CMakeLists.txt

```diff
@@ -151,6 +151,34 @@ add_library(cudf
     src/ast/transform.cu
     src/binaryop/binaryop.cpp
     src/binaryop/compiled/binary_ops.cu
+    src/binaryop/compiled/Add.cu
```

Comment:
IMO, I recommend not to have too many small `*.cu` files like this. As I see below, these files are just for explicit instantiation of template classes. Thus, I would rather want to have them combine into one `.cu` file.

Reply:
Each operator is in separate `*.cu` files to allow parallel build.